planet_explorer/gui/pe_explorer_dockwidget.py

```diff
@@ -180,13 +181,14 @@ def login(self, api_key=None):
             self.p_client.log_in(
                 self.leUser.text(), self.lePass.text(), api_key=api_key
             )
-            self.p_client.blockSignals(False)
         except LoginException as e:
             self.show_message(
                 "Login failed!", show_more=str(e.__cause__), level=Qgis.Warning
             )
             # Stay on login panel if error
             return
+        finally:
+            self.p_client.blockSignals(False)
```

Comment:
@volaya I changed this because if we hit an exception when logging in, the signals will remained blocked from the `p_client`. This was causing test failures after the negative login test (couldn't login after that test).  

Please let me know if this was intended.

Reply:
Ok great - thanks for confirming!